packages/core/src/core.ts

```diff
@@ -91,6 +91,27 @@ export function getInput(name: string, options?: InputOptions): string {
   return val.trim()
 }
 
+/**
+ * Gets the input value of the boolean type in the YAML 1.2 "core schema" specification.
+ * The return value is also in boolean type.
+ * ref: https://yaml.org/spec/1.2/spec.html#id2804923
+ *
+ * @param     name     name of the input to get
+ * @returns   boolean
+ */
+export function getBooleanInput(name: string): boolean {
```

Comment:
```suggestion
export function getBooleanInput(name: string, options?: InputOptions: boolean) {
```

Lets keep consistent with the `getInput` function. We really shouldn't throw if the value does not exist or is invalid, just return false. We can debug if the parsing does not match either case, to help with troubleshooting if users encounter this issue

We can add another option to `InputOptions` to set a default value, and return that instead of false if needed as well. I'm happy to tackle that in another pr if you want.